doc-content/enterprise-only/openshift/freeform-server-managed-deploy-assy.adoc

```diff
@@ -1,7 +1,8 @@
 [id='freeform-server-managed-deploy-assy']
 = Deploying an additional managed {KIE_SERVER} instance for a freeform environment
```

Comment:
Should we remove the wording '**instance** for a freeform environment' consistently?